### PR TITLE
Refactor processing of named graphs provided by the TCK

### PR DESCRIPTION
**Pull request by boggle**

### CODE REVIEW

**Comment by boggle** [comment]
Pending https://github.com/opencypher/openCypher/pull/78

File: community/cypher/compatibility-suite/src/test/java/cypher/FeatureSuiteTest.java [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
+@RunWith( UnpackedResources.class )
 public class FeatureSuiteTest
 {
 
     // These two constants are only used to make testing and debugging easier.
     // If you want to run only a single feature, put the name of the feature file in `FEATURE_TO_RUN` (including .feature)
     // If you want to run only a single scenario, put (part of) its name in the `SCENARIO_NAME_REQUIRED` constant
     // Do not forget to clear these strings to empty strings before you commit!!
-    public static final String FEATURE_TO_RUN = "";
+    public static final String FEATURE_TO_RUN = "CineastsDependent.feature";
```
Comment (Mats-SX) [inline community/cypher/compatibility-suite/src/test/java/cypher/FeatureSuiteTest.java]: Nope!

**Comment by Mats-SX** [comment]
78 was closed; this is now pending https://github.com/opencypher/openCypher/pull/79

File: community/cypher/compatibility-suite/src/test/java/cypher/UnpackedResources.java [hunk 2]
```diff
@@ -85,8 +85,9 @@ private static void findAndUnpackTo( FileSystem sourceFileSystem, String sourceP
             if ( matcher.matches( next ) )
             {
                 File target = new File( targetDirectory, next.getFileName().toString() );
-//                Files.copy( next, target.toPath(), StandardCopyOption.REPLACE_EXISTING );
-                Files.copy( next, target.toPath() );
+                Files.copy( next, target.toPath()
+                        , StandardCopyOption.REPLACE_EXISTING
```
Comment (Mats-SX) [inline community/cypher/compatibility-suite/src/test/java/cypher/UnpackedResources.java]: I don't think this should be the default.

**Pull request merged**